sop.md

```diff
@@ -8,7 +8,7 @@ The sergeant-at-arms (SAA) team consists of the sergeants-at-arms and the IETF C
 
 The sergeants-at-arms strive to monitor all postings on ietf@ietf.org on a daily basis. The IETF Chair also monitors the list but may not be in a position to read each message or monitor every day.
 
-When a member of the SAA team sees a list posting that may be considered inappropriate according to [RFC 3005] and the [Unprofessional Commentary Description], the team member reports it to saa@ietf.org. Subscribers to ietf@ietf.org are also encouraged to report postings they believe to be inappropriate under those guidelines to saa@ietf.org. 
+When IETF discussion list subscribers believe that a posting that may be considered inappropriate according to [RFC 3005] or the [Unprofessional Commentary Description], they are encouraged to report it to saa@ietf.org. Please note that SAA team members are also subscribers to the IETF discussion list and they follow the same process to report posts that they consider inappropriate.
```

Comment:
I think we need to say RFC 3005 and the Unprofessional Commentary Description (not "or"). The two documents work together.

Reply:
I think that commonly "and" means "all", where commonly "or" means "any".  As worded right now, to me I can see an interpretation of "the posting as to be inappropriate according to both/all sources, or it's considered appropriate", which is not the desired outcome.

If not "or", then how about "and possibly"?

Reply:
I think "or" is better than "and possibly." I can live with "or."